cpp/examples/basic_io.cpp

```diff
@@ -195,39 +197,33 @@ int main()
       check(statuses.empty());
       cout << "Batch canceling of all 4 operations" << endl;
     }
-  }
 
-  cout << "stream : " << kvikio::is_stream_available() << endl;
-  if (kvikio::is_stream_available()) {
     {
-      cout << "Performing stream I/O using file handle" << endl;
-      off_t f_off = 0, d_off = 0;
-      ssize_t bytes_done;
-      CUstream stream;
+      cout << "Performing async I/O using file handle" << endl;
+      off_t f_off{0};
+      off_t d_off{0};
+      ssize_t bytes_done{0};
```

Comment:
I think this is incorrect, per the documentation for [`cuFileWriteAsync`](https://docs.nvidia.com/gpudirect-storage/api-reference-guide/index.html#cufilewriteasync):

> The bytes_written pointer should be allocated with `cuMemHostAlloc` or registered with `cuMemHostRegister`, and the pointer to access that memory from the device can be obtained by using `cuMemHostGetDevicePointer`. 

Which we are not doing here.

Reply:
Good catch, fixed